src/lib/ngsiNotify/senderThread.h

```diff
@@ -2,47 +2,63 @@
 #define SENDERTHREAD_H
```

Comment:
Ah wait, I have touched this file also - I added Format to the SenderThreadParams struct !